docs/index.html

```diff
@@ -3,7 +3,7 @@
 
 <head>
 	<meta charset="UTF-8">
-	<title>Microsoft MakeCode</title>
+	<title>Microsoft MakeCode - Home Page</title>
```

Comment:
"Microsoft MakeCode - Documentation Home Page" ? Because otherwise, it's the home page of what?